crafters/nlp/Sentencizer/__init__.py

```diff
@@ -42,7 +42,7 @@ def craft(self, text: str, *args, **kwargs) -> List[Dict]:
         Split the text into sentences.
 
         :param text: the raw text
-        :return: a list of chunk dicts with the cropped images
```

Comment:
it returns a list of chunk dicts.

(if it is images or sentences will depend on the subclass)